.github/workflows/javascript.yml

```
@@ -21,3 +21,7 @@ jobs:
         working-directory: src/dispatch/static/dispatch
         run: |
           npm run lint
+      - name: Run Vitest
+        working-directory: src/dispatch/static/dispatch
+        run: |
+          npm run test:coverage
```

```suggestion
          npm run test
```

Try running w/o coverage to resolve error.